### PR TITLE
Update sliderpane for accessiblity

### PR DESCRIPTION
**Pull request by elioschmutz**
- Update sliderpane for 
  - Show always empty image alt and title attributes on a pane
  - Remove unused descriptionfield on a pane

### CODE REVIEW

File: docs/HISTORY.txt [hunk 1]
```diff
@@ -5,7 +5,10 @@ Changelog
 2.3.1 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Update sliderpane for accessiblity
+  - Show always empty image alt and title attributes on a pane
+  - Remove unused descriptionfield on a pane
+  [elioschmutz]
```
Comment (jone) [inline docs/HISTORY.txt]: I think this is not `rest` valid, can you test that with `longtest` (`zest.releaser`) ?

Comment (elioschmutz) [inline docs/HISTORY.txt]: it works with longtest...

Comment (jone) [inline docs/HISTORY.txt]: ok, but it does not do what you want:
bildschirmfoto 2015-10-30 um 09 58 34

Comment (elioschmutz) [inline docs/HISTORY.txt]: :+1: you're right... now it should be good:

bildschirmfoto 2015-10-30 um 10 08 18

**Pull request merged**